.circleci/config.yml

```diff
@@ -363,36 +402,54 @@ commands:
                   --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
                   --num-uniform-shards 3
               else
-                gcloud beta firebase test android run --type instrumentation \
-                --app << parameters.app_target >>/build/outputs/apk/debug/<< parameters.app_target >>-debug.apk \
-                --test << parameters.module_target >>/build/outputs/apk/androidTest/debug/<< parameters.module_target >>-debug-androidTest.apk \
-                --device model=harpia,version=23,locale=en,orientation=portrait \
-                --device model=blueline,version=28,locale=en,orientation=portrait \
-                --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
-                --num-uniform-shards 4
+                if << parameters.nightly >> ; then
+                  gcloud beta firebase test android run --type instrumentation \
+                    --app << parameters.app_target >>/build/outputs/apk/debug/<< parameters.app_target >>-debug.apk \
+                    --test << parameters.module_target >>/build/outputs/apk/androidTest/debug/<< parameters.module_target >>-debug-androidTest.apk \
+                    --device model=harpia,version=23,locale=en,orientation=portrait \
+                    --device model=blueline,version=28,locale=en,orientation=portrait \
+                    --device model=redfin,version=30,locale=en,orientation=portrait \
+                    --device model=oriole,version=33,locale=en,orientation=portrait \
+                    --timeout 45m --use-orchestrator --no-record-video --num-flaky-test-attempts 1 \
+                    --num-uniform-shards 4
+                else
+                  gcloud beta firebase test android run --type instrumentation \
+                    --app << parameters.app_target >>/build/outputs/apk/debug/<< parameters.app_target >>-debug.apk \
+                    --test << parameters.module_target >>/build/outputs/apk/androidTest/debug/<< parameters.module_target >>-debug-androidTest.apk \
+                    --device model=redfin,version=30,locale=en,orientation=portrait \
```

Comment:
Thinking that it might be useful to launch PR tests on the latest available SDK version, wdyt?

Reply:
The redfin device is currently the "default" device which means it will have the highest availability and I'm hoping it's also the most stable. We will check the latest available Android version on a weekly basis and can use this system manually when we bump upstream dependencies

Reply:
My idea was that typically the issues happen with the newest or oldest devices, using latest would allow to catch them as soon as the buggy code gets merged. But I guess we still should be fine with the weekly tests.